Ex_19/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const birthMonth = "December";
+const firstName = "Natali";
+const LastName = "Fox";
+let city;
+city = "Winnipeg";
+let age;
+age = 25;
+let nextCity;
+nextCity = city;
+
+console.log('City: ', city);
+console.log('Age: ', age);
+console.log('Next City: ', nextCity);
+alert (`Customer ${firstName}, ${LastName}`);
```

Comment:
For variable names try using camelCase.
PascalCase is mostly used on class names in Javascript.
(You can research what is camelCase and PascalCase, it's interesting ;) )